### ISSUE TITLE
A comment in fontawesome-svg-core causes bug when inlined

### ISSUE THREAD

**Issue by bengladwell** [FortAwesome/Font-Awesome#18473]
**Describe the bug**
[This comment](https://github.com/FortAwesome/Font-Awesome/blob/7d3d774145ac38663f6d1effc6def0334b68ab7e/js-packages/%40fortawesome/fontawesome-svg-core/index.es.js#L235) in fontawesome-svg-core/index.es.js contains a script tag. In development builds in which code is inlined directly on the page this breaks the page. The browser sees `<script>` and tries to interpret this as a real script tag.

I realize it is unusual to inline a built JS asset like this, but my use case requires it.

**To Reproduce**

**Expected behavior**
If this comment is not needed, the easiest solution would be to remove it.

**Screenshots**

**Version and implementation**
Version: fontawesome-svg-core@1.2.32

- [ ] SVG with JS
- [ ] Web Fonts with CSS
- [ ] SVG Sprites
- [ ] On the Desktop

**Bug report checklist**
- [ ] I have filled out as much of the above information as I can
- [ ] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [ ] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by tagliala** [comment]
@robmadole any chance to take a look? If this comment is breaking a use case we may not use valid html syntax

**Comment by robmadole** [comment]
@bengladwell yep. I've been there and understand that strange `<script>` tag in a comment thing.

We can remove it. We'll get this released in both 5.x and 6.x but it will be awhile as we are lazer-focused on releasing 6 ATM.

**Comment by bengladwell** [comment]
@tagliala @robmadole thanks!

**Comment by willemvb** [comment]
@bengladwell Had the same issue. In case it helps, this workaround describes a way to HTML-escape the `</script>` tag in the JS comment.
https://stackoverflow.com/a/32861662/1716919

**Comment by bengladwell** [comment]
Thanks @willemvb!

**Comment by bengladwell** [comment]
FWIW I am doing this as a `prebuild` package.json script command:
```bash
perl -p -i -e 's/<script.+<\\/script>//g' node_modules/@fortawesome/fontawesome-svg-core/index.es.js
```

**Comment by AlexVanderbist** [comment]
Ended up doing the same thing but using `sed` instead:
```
    "postinstall": "sed -i '' '/For example <script/d' ./node_modules/@fortawesome/fontawesome-svg-core/index.es.js"
```
